tests/integration/src/test/scala/mesosphere/marathon/integration/LaunchQueueIntegrationTest.scala

```diff
@@ -39,10 +39,14 @@ class LaunchQueueIntegrationTest extends AkkaIntegrationTest with EmbeddedMarath
       val response = marathon.launchQueue()
       response should be(OK)
 
-      val queue = response.value.queue
-      queue should have size 1
-      queue.head.app.id.toPath should be (appId)
-      queue.head.count should be (5)
+      // The LaunchQueue will currently only process one instance add at a time
+      // Therefore, the LaunchQueue will only eventually report 5 queued instances
+      WaitTestSupport.waitUntil("All 5 instance show up on the queue", 30.seconds) {
+        val queue = response.value.queue
+        queue should have size 1
+        queue.head.app.id.toPath should be (appId)
+        queue.head.count == 5
+      }
```

Comment:
Please use

```scala
eventually {
  queue should have size 1
  queue.head.app.id.toPath should be (appId)
  queue.head.count should be(5)
}
```

Reply:
Where is the change? I think I'm looking at an outdated diff. Or do you reject my comment, @alenkacz? That would be fine but I'd like to understand why should prefer ` WaitTestSupport.waitUntil` over `eventually`.

Reply:
![screen shot 2018-11-16 at 09 00 33](https://user-images.githubusercontent.com/2418440/48605831-2b83a900-e97e-11e8-927e-124d0da1042a.png)

This is what I see. The page is reloaded.

Reply:
Weird. I had to reload the Github page several times. Now it is there. Sorry for the confusion.